tests/bluetooth/bsim_bt/bsim_test_advx/src/main.c

```diff
@@ -247,6 +247,80 @@ static void test_advx_main(void)
 	}
 	printk("success.\n");
 
+	k_sleep(K_MSEC(400));
```

Comment:
Do not need delays when there is no on-air activity that needs analysis.

Reply:
Fixed.